### PR TITLE
Feature std dev

### PR DESCRIPTION
**Pull request by 0rC0**

### CODE REVIEW

File: inflammation-analysis.py
Comment (chienc1) [inline inflammation-analysis.py:1]: last view_data key is missing?

File: inflammation/models.py [hunk 1]
```diff
@@ -32,3 +32,13 @@ def daily_min(data):
     """Calculate the daily min of a 2d inflammation data array."""
     return np.min(data, axis=0)
 
+
+def s_dev(data):
+    """Computes and returns standard deviation for data."""
+    mmm = np.mean(data, axis=0)
```
Comment (0rC0) [inline inflammation/models.py]: here you could use i.e. np.std.

i.e. def s_dev(data): return np.std(data, axis=0)

Comment (0rC0) [reply inflammation/models.py:38]: last view_data key is missing?
test to update

File: inflammation/models.py
Comment (chienc1) [inline inflammation/models.py:1]: single comment

File: inflammation/models.py (hunk 1 above)
Comment (chienc1) [reply inflammation/models.py]: done

**Pull request merged**